jenkins/Jenkinsfile-full-deploy-nightly

```diff
@@ -0,0 +1,43 @@
+pipeline {
+  agent any
+
+  environment {
+    DOWNLOAD_AREA = '/home/data/httpd/download.eclipse.org/modeling/tmf/xtext/monorepo'
+    REPOSITORY_PATH="${DOWNLOAD_AREA}/updates/nightly"
+  }
+
+  tools {
+    maven "apache-maven-3.8.6"
```

Comment:
is the .6 intentional?

Reply:
Well, I copied that part from the current Jenkinsfiles, e.g., https://github.com/eclipse/xtext-core/blob/master/Jenkinsfile#L32

should I change that with a more recent version?

Reply:
keep it for now. we can update later